app/models/wine_region.rb

```diff
@@ -0,0 +1,7 @@
+class WineRegion < ApplicationRecord
+  validates_presence_of :name
+  belongs_to :country
+  belongs_to :state
```

Comment:
needs optional: true since rails 5 now requires this association to be set by default. It looks like shoulda matchers doesn't support this yet, maybe we submit a pr?
https://github.com/thoughtbot/shoulda-matchers/issues/870